src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const listPopulation = document.querySelectorAll('.population');
+const arrayPopulation = [...listPopulation].map(
+  item => item.innerText.replace(/,/g, '')
+);
+const numberPopulation = arrayPopulation.map(item => Number(item));
+
+const totalAmount = numberPopulation.reduce((sum, current) => sum + current);
+const total = String(totalAmount).replace(/(\d)(?=(\d\d\d)+([^\d]|$))/g, '$1,');
```

Comment:
remove this, use toLocaleString method instead
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString